README.md

```diff
@@ -7,8 +7,7 @@
 > A curated list of awesome CircuitPython guides, videos, libraries, frameworks, software and resources.
 
 
-
-https://user-images.githubusercontent.com/1685947/115119719-d6e21f00-9f77-11eb-84bf-3f7af59948a3.mov
+<video style="display: block; width: 720px; margin: auto;" controls src="https://user-images.githubusercontent.com/1685947/115119719-d6e21f00-9f77-11eb-84bf-3f7af59948a3.mov"></video>
```

Comment:
Adding a hardcoded `width: 720px;` results in a bit of a broken experience on mobile devices.  The following screenshot is a simulation from Chrome DevTools for a Pixel 2 XL:
![image](https://user-images.githubusercontent.com/37907774/143563060-2f0fc296-bc91-4928-b60f-8a9b93b4c2e2.png)

Perhaps change it do be more responsive and do something like `max-width: 720px; width: 100%`.

A bit unrelated to this pull request, but in the responsive vein on line 2, the Blinka logo could also be made responsive on mobile by adding a `style="width: 100%; max-width: 400px"` to the `<img>` tag:
![image](https://user-images.githubusercontent.com/37907774/143564224-d70000ae-81ce-4bda-b263-d80742f55a83.png)